tests/modules/services/emacs/emacs-socket-and-startWithUserSession.nix

```
@@ -0,0 +1,29 @@
+{ config, lib, pkgs, ... }:
```

Since `config` is unused:
```suggestion
{ lib, pkgs, ... }:
```

---

fixed. I also removed `pkgs`